src/aat/resources/features/F-1005 - GlobalSearch - Search cases/S-1005.10.td.json

```diff
@@ -79,11 +86,11 @@
             "FirstNameValue"
           ],
           "caseNameHmctsInternal": "Name Internal",
-          "baseLocationId": "1",
+          "baseLocationId": "${[scenarioContext][testData][request][body][searchCriteria][caseManagementBaseLocationIds][0]}",
```

Comment:
My only suggestion would be to leave the old values for this one, as the callback is not changed and the values are static. It would make it a bit more readable, in my view.

Reply:
resolved in fb506344bf94aaebeabf6903aab21df062468b0d